dash_cytoscape/CyLeaflet.py

```diff
@@ -64,7 +64,11 @@ def __init__(
             s: {"id": id, "component": "cyleaflet", "sub": s}
             for s in ["cy", "leaf", "elements"]
         }
-        self.ids["component"] = "leaflet"
+
+        self.CYTOSCAPE_ID = self.ids["cy"]
+        self.LEAFLET_ID = self.ids["leaf"]
+        self.ELEMENTS_ID = self.ids["elements"]
```

Comment:
Just to make it easier than `cyleaflet_instance.ids["leaf"]`? I guess this way you get autocompletion?

Regardless, in order to use this you needed to pull `cyleaflet_instance` out of the layout... which might not always be possible, I wonder if it would also be worth making a `cyto.CyLeaflet.get_leaflet_id(cyleaflet_id)` etc that would construct the PMC ID for you - and then you could even call `cyto.CyLeaflet.get_leaflet_id(ALL)` or `(MATCH)` if you wanted to construct your own PMC?

Reply:
Yeah exactly, this way you get autocomplete and it looks cleaner than accessing a dictionary.

I see your point -- I'm open to switching this to exposing static functions instead.

Reply:
I don't think it needs to be one or the other - this pattern is very nice when it works, but we can add the functions as well to cover the cases when it doesn't work.